app-crypt/tpm-emulator/files/tpm-emulator-0.7.5-static-libs.patch

```diff
@@ -0,0 +1,11 @@
+https://bugs.gentoo.org/726492
+--- a/tddl/CMakeLists.txt
++++ b/tddl/CMakeLists.txt
+@@ -16,7 +16,6 @@ elseif(WIN32)
+ endif()
+ 
+ install(TARGETS tddl-tpm-emulator DESTINATION ${CMAKE_INSTALL_LIBDIR})
+-install(TARGETS tddl-tpm-emulator_static DESTINATION ${CMAKE_INSTALL_LIBDIR})
```

Comment:
can you completely remove the `tddl-tpm-emulator_static` target too?

Reply:
Sure